aws-sdk-core/lib/aws-sdk-core/api/customizations.rb

```diff
@@ -157,6 +157,24 @@ def apply_plugins(client_class)
             end
           end
         end
+
+        api['shapes']['ExpiresString'] = { 'type' => 'string' }
+        %w(HeadObjectOutput GetObjectOutput).each do |shape|
+          members = api['shapes'][shape]['members']
+          # inject this into sort order so this appears directly
+          # after #expires
```

Comment:
Technically this will make it show up BEFORE 'Expires' in my testing, not after.